### PR TITLE
Add "light-brand" and "dark-brand" colors (add a second color picker)

### LINKED ISSUE #1953: add "light-brand" and "dark-brand" colors (add a second color picker)
defaults:

light: `#212121`
dark: `#fafafa`

image

### PR DESCRIPTION
**Pull request by alishaz-polymath**
## What does this PR do?

- Adds a 2nd color picker for dark(theme)-brand color
- Converts the existing brand color to be used as light(theme)-brand color

Fixes #1953 

## Type of change

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update

## How should this be tested?

- [x] Update the respective theme brand colors and view the pages in those themes (light and dark)

## Checklist

- I haven't read the [contributing guide](https://github.com/calcom/cal.com/blob/main/CONTRIBUTING.md)
- My code doesn't follow the style guidelines of this project
- I haven't performed a self-review of my own code and corrected any misspellings
- I haven't commented my code, particularly in hard-to-understand areas
- I haven't checked if my PR needs changes to the documentation
- I haven't checked if my changes generate no new warnings
- I haven't added tests that prove my fix is effective or that my feature works
- I haven't checked if new and existing unit tests pass locally with my changes

### CODE REVIEW

**Review by alannnc — APPROVED** [review]
👍

File: apps/web/components/AddToHomescreen.tsx [hunk 1]
```diff
@@ -18,7 +18,7 @@ export default function AddToHomescreen() {
         <div className="rounded-lg p-2 shadow-lg sm:p-3" style={{ background: "#2F333D" }}>
           <div className="flex flex-wrap items-center justify-between">
             <div className="flex w-0 flex-1 items-center">
-              <span className="bg-brand text-brandcontrast flex rounded-lg bg-opacity-30 p-2">
+              <span className="bg-brand text-brandcontrast dark:bg-darkmodebrand dark:text-darkmodebrandcontrast flex rounded-lg bg-opacity-30 p-2">
```
Comment (PeerRich) [inline apps/web/components/AddToHomescreen.tsx:21]: NIT: `dark:bg-darkmodebrand dark:text-darkmodebrandcontrast` can we shorten this?  lets get rid of `mode`

File: apps/web/components/Shell.tsx [hunk 2]
```diff
@@ -192,7 +192,7 @@ export default function Shell(props: {
   }
   return (
     <>
-      <CustomBranding val={user?.brandColor} />
+      <CustomBranding lightVal={user?.brandColor} darkVal={user?.darkBrandColor} />
```
Comment (PeerRich) [inline apps/web/components/Shell.tsx:195]: NIT: this could be `light` and `dark` only, no need for `Val`

**Review by PeerRich — APPROVED** [review]

**Comment by PeerRich** [comment]
@alishaz-polymath just making sure, default values are #292929 and #fafafa ?

**Comment by alishaz-polymath** [comment]
> @alishaz-polymath just making sure, default values are #292929 and #fafafa ?

Yes, that's correct.

**Pull request merged**